src/Options.cpp

```diff
@@ -536,7 +530,7 @@ bool Options::parseArg(int key, const char *arg)
         return parseArg(key, p ? strtoull(p, nullptr, 16) : strtoull(arg, nullptr, 10));
     }
 
-    case 4013: { /* --doublehash-thread-mask */
+    case 4013: { /* --multihash-thread-mask */
```

Comment:
To be backwards compatible to old doublehash param i'll add it here.

Reply:
Now, I just map both strings to 4013. This should serve the purpose.

```
{ "doublehash-thread-mask",     1, nullptr, 4013 },
{ "multihash-thread-mask",     1, nullptr, 4013 },
```